sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -111,9 +111,7 @@ export function record(
 
   return {
     stop: function() {
-      if (isRecordMode()) {
-        recorder.stop();
-      }
+      recorder.stop();
```

Comment:
Moved the if-block to the lower level `stop()` method present in `NockRecorder` and `NiseRecorder` to comfortably add cleanup code during playback when the recorder is stopped in the test.